internal/compiler/widgets/fluent-base/tableview.slint

```diff
@@ -201,48 +203,42 @@ export component StandardTableView {
             }
         }
 
-        i-scroll-view := ScrollView {
-            vertical-stretch: 1;
```

Comment:
Note to self: I guess we should keep the `vertical-stretch: 1;`.

Reply:
Yes it's true. It can be removed.